keyboards/wuque/epiphany70/info.json

```
@@ -0,0 +1,690 @@
+{
+    "keyboard_name": "epiphany70",
+    "manufacturer": "wuque studio",
+    "maintainer": "spbgzh",
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+      "bootmagic": true,
+      "command": false,
+      "console": false,
+      "extrakey": true,
+      "mousekey": true,
+      "nkro": true
+    },
+    "matrix_pins": {
+      "rows": ["B3", "B2", "F0", "D3", "D1"],
+      "cols": ["E6", "D4", "D7", "F7", "D2", "D5", "D6", "F5", "F4", "F1", "C7", "C6", "B6", "B5", "B4", "B7", "B1", "B0"]
+    },
+    "usb": {
+      "vid": "0x918D",
+      "pid": "0x0002",
+      "device_version": "0.0.1"
+    },
+    "indicators": {
+      "caps_lock": "F6",
+      "on_state": 0
+    },
+    "ws2812": {
+      "pin": "D0"
+    },
+    "rgblight": {
+      "max_brightness": 200
+    },
```

rbglight defs migrated from `config.h`
```suggestion
        "led_count": 7,
        "sleep": true,
        "saturation_steps": 8,
        "brightness_steps": 8,
        "animations": {
            "breathing": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "snake": true,
            "knight": true,
            "christmas": true,
            "static_gradient": true,
            "rgb_test": true,
            "alternating": true,
            "twinkle": true
        }
    },
```

---

I just dont know how to write it(